src/main-process/modules/tray.js

```diff
@@ -1,14 +1,31 @@
-'use strict';
+/**
+ * Module in charge of the Tray
+ */
 
 const os = require('os');
+const path = require('path');
+const { Tray, Menu, app, ipcMain, nativeImage } = require('electron');
+
+const ModuleWindow = require('./module-window');
+const { IPC_PLAYER_ACTION } = require('../../shared/constants/ipc');
+
+
+class TrayManager extends ModuleWindow {
+  constructor(window) {
+    super(window);
+  }
+
+  load() {
+    const appRoot = path.resolve(__dirname, '../../..'); // Maybe a better way to know this?
```

Comment:
Not sure how much better that would be, but we might have a `paths` module with all commonly accessed paths pre-computed.